tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -12,22 +12,18 @@
 import logging
 import time
 
-import boto3
 import pytest
 from assertpy import assert_that
 from remote_command_executor import RemoteCommandExecutionError, RemoteCommandExecutor
 from retrying import retry
 from time_utils import minutes, seconds
 
-from tests.common.assertions import assert_asg_desired_capacity, assert_no_errors_in_logs, assert_scaling_worked
+from tests.common.assertions import assert_no_errors_in_logs, assert_no_node_in_ec2, assert_scaling_worked
 from tests.common.mpi_common import OS_TO_ARCHITECTURE_TO_OPENMPI_MODULE, compile_mpi_ring
 from tests.common.schedulers_common import SlurmCommands
-from tests.schedulers.common import assert_overscaling_when_job_submitted_during_scaledown
 
 
-@pytest.mark.skip(reason="FIXME")
 @pytest.mark.regions(["us-east-2"])
-@pytest.mark.instances(["c5.xlarge", "m6g.xlarge"])
 @pytest.mark.schedulers(["slurm"])
 @pytest.mark.usefixtures("instance", "scheduler")
```

Comment:
either we remove the instance fixture or we keep the instances marker for the head node. Otherwise it'll loop on all instances.

Reply:
Done